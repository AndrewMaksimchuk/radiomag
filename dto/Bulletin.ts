export interface Bulletin {
  id?: number | undefined;
  email: string;
}
