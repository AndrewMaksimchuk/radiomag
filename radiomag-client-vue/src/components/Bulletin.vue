<template>
  <article class="linecard borders">
    <h2 class="linecard__header-text">Бюлетень</h2>
    <form class="linecard__form" action="#">
      <label class="linecard__form-label" for="email"
        >Інформація про події, акції та новий товар прямо у ваш кошик.</label
      >
      <input
        type="email"
        required
        name="email"
        id="email"
        class="linecard__form-select bulletin__input"
        placeholder="Напишіть ваший E-mail"
      />
      <button class="linecard__form-submit cursor-pointer" type="submit">Записатися</button>
    </form>
  </article>
</template>

<style lang="scss">

.bulletin__input {
  font-size: 1.3rem;
  padding: 5px;

  &:hover {
    cursor: text;
  }
}

</style>
