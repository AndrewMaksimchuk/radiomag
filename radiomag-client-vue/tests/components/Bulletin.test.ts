import { describe, it, expect } from "vitest";
import { shallowMount } from "@vue/test-utils";
import Bulletin from "@/components/BulletinComponent.vue";

const wrapper = shallowMount(Bulletin, {
  global: {
    mocks: {
      $t: (text: string) => text,
    },
  },
});

describe("Bulletin component", () => {
  it("should be displayed", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
