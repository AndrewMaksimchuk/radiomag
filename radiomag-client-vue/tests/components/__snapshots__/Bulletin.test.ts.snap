// Vitest Snapshot v1

exports[`Bulletin component > should be displayed 1`] = `
VueWrapper {
  "__app": {
    "_component": {
      "name": "VTU_ROOT",
      "render": [Function],
      "setup": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <article
        class="linecard borders m-hide"
        data-v-72aaa15f=""
      >
        <h2
          class="linecard__header-text"
          data-v-72aaa15f=""
        >
          bulletin.header
        </h2>
        <form
          action="#"
          class="linecard__form"
          data-v-72aaa15f=""
        >
          <label
            class="linecard__form-label"
            data-v-72aaa15f=""
            for="email"
          >
            bulletin.label
          </label>
          <input
            class="linecard__form-select bulletin__input"
            data-v-72aaa15f=""
            id="email"
            name="email"
            placeholder="bulletin.placeholder"
            required=""
            type="email"
          />
          <button
            class="linecard__form-submit cursor-pointer"
            data-v-72aaa15f=""
            type="submit"
          >
            bulletin.submit
          </button>
        </form>
      </article>
    </div>,
    "_context": {
      "app": [Circular],
      "components": {
        "transition": {
          "name": "transition",
        },
        "transition-group": {
          "name": "transition-group",
        },
      },
      "config": {
        "compilerOptions": {},
        "errorHandler": undefined,
        "globalProperties": {},
        "isNativeTag": [Function],
        "optionMergeStrategies": {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": {},
      "emitsCache": WeakMap {},
      "mixins": [
        {
          "beforeCreate": [Function],
        },
      ],
      "optionsCache": WeakMap {},
      "propsCache": WeakMap {},
      "provides": {},
      "reload": [Function],
    },
    "_instance": {
      "a": null,
      "accessCache": {},
      "appContext": {
        "app": [Circular],
        "components": {
          "transition": {
            "name": "transition",
          },
          "transition-group": {
            "name": "transition-group",
          },
        },
        "config": {
          "compilerOptions": {},
          "errorHandler": undefined,
          "globalProperties": {},
          "isNativeTag": [Function],
          "optionMergeStrategies": {},
          "performance": false,
          "warnHandler": undefined,
        },
        "directives": {},
        "emitsCache": WeakMap {},
        "mixins": [
          {
            "beforeCreate": [Function],
          },
        ],
        "optionsCache": WeakMap {},
        "propsCache": WeakMap {},
        "provides": {},
        "reload": [Function],
      },
      "asyncDep": null,
      "asyncResolved": false,
      "attrs": {},
      "bc": null,
      "bm": null,
      "bu": null,
      "bum": null,
      "c": null,
      "components": null,
      "ctx": {
        "$t": [Function],
        "VTU_COMPONENT": {
          "$t": [Function],
        },
      },
      "da": null,
      "data": {},
      "devtoolsRawSetupState": {
        "VTU_COMPONENT": RefImpl {
          "__v_isRef": true,
          "__v_isShallow": false,
          "_rawValue": {
            "$t": [Function],
          },
          "_value": {
            "$t": [Function],
          },
          "dep": undefined,
        },
      },
      "directives": null,
      "ec": null,
      "effect": ReactiveEffect {
        "active": true,
        "allowRecurse": true,
        "deps": [
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
          Set {
            [Circular],
          },
        ],
        "fn": [Function],
        "onTrack": undefined,
        "onTrigger": undefined,
        "parent": undefined,
        "scheduler": [Function],
      },
      "emit": [Function],
      "emitsOptions": null,
      "emitted": null,
      "exposeProxy": null,
      "exposed": null,
      "inheritAttrs": undefined,
      "isDeactivated": false,
      "isMounted": true,
      "isUnmounted": false,
      "m": null,
      "next": null,
      "parent": null,
      "props": {},
      "propsDefaults": {},
      "propsOptions": [
        {},
        [],
      ],
      "provides": {},
      "proxy": {
        "$t": [Function],
        "VTU_COMPONENT": {
          "$t": [Function],
        },
      },
      "refs": {
        "VTU_COMPONENT": {
          "$t": [Function],
        },
      },
      "render": [Function],
      "renderCache": [],
      "root": [Circular],
      "rtc": null,
      "rtg": null,
      "scope": EffectScope {
        "_active": true,
        "cleanups": [],
        "detached": true,
        "effects": [
          ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": [
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
              Set {
                [Circular],
              },
            ],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
        ],
        "parent": undefined,
      },
      "setupContext": null,
      "setupState": {
        "VTU_COMPONENT": {
          "$t": [Function],
        },
      },
      "slots": {},
      "sp": null,
      "subTree": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": null,
        "children": null,
        "component": {
          "a": null,
          "accessCache": {
            "$t": 4,
            "@@__IMMUTABLE_ITERABLE__@@": 0,
            "@@__IMMUTABLE_RECORD__@@": 0,
            "__v_isReadonly": 0,
            "__v_isRef": 0,
            "__v_isShallow": 0,
            "__v_raw": 0,
            "__v_skip": 4,
            "_isMockFunction": 0,
            "constructor": 0,
            "toJSON": 0,
            Symbol(Symbol.toStringTag): 0,
          },
          "appContext": {
            "app": [Circular],
            "components": {
              "transition": {
                "name": "transition",
              },
              "transition-group": {
                "name": "transition-group",
              },
            },
            "config": {
              "compilerOptions": {},
              "errorHandler": undefined,
              "globalProperties": {},
              "isNativeTag": [Function],
              "optionMergeStrategies": {},
              "performance": false,
              "warnHandler": undefined,
            },
            "directives": {},
            "emitsCache": WeakMap {},
            "mixins": [
              {
                "beforeCreate": [Function],
              },
            ],
            "optionsCache": WeakMap {},
            "propsCache": WeakMap {},
            "provides": {},
            "reload": [Function],
          },
          "asyncDep": null,
          "asyncResolved": false,
          "attrs": {},
          "bc": null,
          "bm": null,
          "bu": null,
          "bum": null,
          "c": null,
          "components": {},
          "ctx": {
            "$t": [Function],
          },
          "da": null,
          "data": {},
          "directives": null,
          "ec": null,
          "effect": ReactiveEffect {
            "active": true,
            "allowRecurse": true,
            "deps": [],
            "fn": [Function],
            "onTrack": undefined,
            "onTrigger": undefined,
            "parent": undefined,
            "scheduler": [Function],
          },
          "emit": [Function],
          "emitsOptions": null,
          "emitted": null,
          "exposeProxy": null,
          "exposed": null,
          "inheritAttrs": undefined,
          "isDeactivated": false,
          "isMounted": true,
          "isUnmounted": false,
          "m": null,
          "next": null,
          "parent": [Circular],
          "props": {},
          "propsDefaults": {},
          "propsOptions": [
            {},
            [],
          ],
          "provides": {},
          "proxy": {
            "$t": [Function],
          },
          "refs": {},
          "render": [Function],
          "renderCache": [],
          "root": [Circular],
          "rtc": null,
          "rtg": null,
          "scope": EffectScope {
            "_active": true,
            "cleanups": [],
            "detached": true,
            "effects": [
              ReactiveEffect {
                "active": true,
                "allowRecurse": true,
                "deps": [],
                "fn": [Function],
                "onTrack": undefined,
                "onTrigger": undefined,
                "parent": undefined,
                "scheduler": [Function],
              },
            ],
            "parent": undefined,
          },
          "setupContext": null,
          "setupState": {},
          "slots": {},
          "sp": null,
          "subTree": {
            "__v_isVNode": true,
            "__v_skip": true,
            "anchor": null,
            "appContext": null,
            "children": [
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": "bulletin.header",
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <h2
                  class="linecard__header-text"
                  data-v-72aaa15f=""
                >
                  bulletin.header
                </h2>,
                "key": null,
                "patchFlag": 1,
                "props": {
                  "class": "linecard__header-text",
                },
                "ref": null,
                "scopeId": "data-v-72aaa15f",
                "shapeFlag": 9,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "h2",
              },
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": [
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": "bulletin.label",
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <label
                      class="linecard__form-label"
                      data-v-72aaa15f=""
                      for="email"
                    >
                      bulletin.label
                    </label>,
                    "key": null,
                    "patchFlag": 1,
                    "props": {
                      "class": "linecard__form-label",
                      "for": "email",
                    },
                    "ref": null,
                    "scopeId": "data-v-72aaa15f",
                    "shapeFlag": 9,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "label",
                  },
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": null,
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": [
                      "placeholder",
                    ],
                    "el": <input
                      class="linecard__form-select bulletin__input"
                      data-v-72aaa15f=""
                      id="email"
                      name="email"
                      placeholder="bulletin.placeholder"
                      required=""
                      type="email"
                    />,
                    "key": null,
                    "patchFlag": 8,
                    "props": {
                      "class": "linecard__form-select bulletin__input",
                      "id": "email",
                      "name": "email",
                      "placeholder": "bulletin.placeholder",
                      "required": "",
                      "type": "email",
                    },
                    "ref": null,
                    "scopeId": "data-v-72aaa15f",
                    "shapeFlag": 1,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "input",
                  },
                  {
                    "__v_isVNode": true,
                    "__v_skip": true,
                    "anchor": null,
                    "appContext": null,
                    "children": "bulletin.submit",
                    "component": null,
                    "ctx": [Circular],
                    "dirs": null,
                    "dynamicChildren": null,
                    "dynamicProps": null,
                    "el": <button
                      class="linecard__form-submit cursor-pointer"
                      data-v-72aaa15f=""
                      type="submit"
                    >
                      bulletin.submit
                    </button>,
                    "key": null,
                    "patchFlag": 1,
                    "props": {
                      "class": "linecard__form-submit cursor-pointer",
                      "type": "submit",
                    },
                    "ref": null,
                    "scopeId": "data-v-72aaa15f",
                    "shapeFlag": 9,
                    "slotScopeIds": null,
                    "ssContent": null,
                    "ssFallback": null,
                    "staticCount": 0,
                    "suspense": null,
                    "target": null,
                    "targetAnchor": null,
                    "transition": null,
                    "type": "button",
                  },
                ],
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <form
                  action="#"
                  class="linecard__form"
                  data-v-72aaa15f=""
                >
                  <label
                    class="linecard__form-label"
                    data-v-72aaa15f=""
                    for="email"
                  >
                    bulletin.label
                  </label>
                  <input
                    class="linecard__form-select bulletin__input"
                    data-v-72aaa15f=""
                    id="email"
                    name="email"
                    placeholder="bulletin.placeholder"
                    required=""
                    type="email"
                  />
                  <button
                    class="linecard__form-submit cursor-pointer"
                    data-v-72aaa15f=""
                    type="submit"
                  >
                    bulletin.submit
                  </button>
                </form>,
                "key": null,
                "patchFlag": 0,
                "props": {
                  "action": "#",
                  "class": "linecard__form",
                },
                "ref": null,
                "scopeId": "data-v-72aaa15f",
                "shapeFlag": 17,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "form",
              },
            ],
            "component": null,
            "ctx": [Circular],
            "dirs": null,
            "dynamicChildren": [
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": "bulletin.header",
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <h2
                  class="linecard__header-text"
                  data-v-72aaa15f=""
                >
                  bulletin.header
                </h2>,
                "key": null,
                "patchFlag": 1,
                "props": {
                  "class": "linecard__header-text",
                },
                "ref": null,
                "scopeId": "data-v-72aaa15f",
                "shapeFlag": 9,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "h2",
              },
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": "bulletin.label",
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <label
                  class="linecard__form-label"
                  data-v-72aaa15f=""
                  for="email"
                >
                  bulletin.label
                </label>,
                "key": null,
                "patchFlag": 1,
                "props": {
                  "class": "linecard__form-label",
                  "for": "email",
                },
                "ref": null,
                "scopeId": "data-v-72aaa15f",
                "shapeFlag": 9,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "label",
              },
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": null,
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": [
                  "placeholder",
                ],
                "el": <input
                  class="linecard__form-select bulletin__input"
                  data-v-72aaa15f=""
                  id="email"
                  name="email"
                  placeholder="bulletin.placeholder"
                  required=""
                  type="email"
                />,
                "key": null,
                "patchFlag": 8,
                "props": {
                  "class": "linecard__form-select bulletin__input",
                  "id": "email",
                  "name": "email",
                  "placeholder": "bulletin.placeholder",
                  "required": "",
                  "type": "email",
                },
                "ref": null,
                "scopeId": "data-v-72aaa15f",
                "shapeFlag": 1,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "input",
              },
              {
                "__v_isVNode": true,
                "__v_skip": true,
                "anchor": null,
                "appContext": null,
                "children": "bulletin.submit",
                "component": null,
                "ctx": [Circular],
                "dirs": null,
                "dynamicChildren": null,
                "dynamicProps": null,
                "el": <button
                  class="linecard__form-submit cursor-pointer"
                  data-v-72aaa15f=""
                  type="submit"
                >
                  bulletin.submit
                </button>,
                "key": null,
                "patchFlag": 1,
                "props": {
                  "class": "linecard__form-submit cursor-pointer",
                  "type": "submit",
                },
                "ref": null,
                "scopeId": "data-v-72aaa15f",
                "shapeFlag": 9,
                "slotScopeIds": null,
                "ssContent": null,
                "ssFallback": null,
                "staticCount": 0,
                "suspense": null,
                "target": null,
                "targetAnchor": null,
                "transition": null,
                "type": "button",
              },
            ],
            "dynamicProps": null,
            "el": <article
              class="linecard borders m-hide"
              data-v-72aaa15f=""
            >
              <h2
                class="linecard__header-text"
                data-v-72aaa15f=""
              >
                bulletin.header
              </h2>
              <form
                action="#"
                class="linecard__form"
                data-v-72aaa15f=""
              >
                <label
                  class="linecard__form-label"
                  data-v-72aaa15f=""
                  for="email"
                >
                  bulletin.label
                </label>
                <input
                  class="linecard__form-select bulletin__input"
                  data-v-72aaa15f=""
                  id="email"
                  name="email"
                  placeholder="bulletin.placeholder"
                  required=""
                  type="email"
                />
                <button
                  class="linecard__form-submit cursor-pointer"
                  data-v-72aaa15f=""
                  type="submit"
                >
                  bulletin.submit
                </button>
              </form>
            </article>,
            "key": null,
            "patchFlag": 0,
            "props": {
              "class": "linecard borders m-hide",
            },
            "ref": null,
            "scopeId": "data-v-72aaa15f",
            "shapeFlag": 17,
            "slotScopeIds": null,
            "ssContent": null,
            "ssFallback": null,
            "staticCount": 0,
            "suspense": null,
            "target": null,
            "targetAnchor": null,
            "transition": null,
            "type": "article",
          },
          "suspense": null,
          "suspenseId": 0,
          "type": {
            "__file": "/home/andrew/Code/radiomag/radiomag-client-vue/src/components/BulletinComponent.vue",
            "__scopeId": "data-v-72aaa15f",
            "components": {},
            "render": [Function],
          },
          "u": null,
          "uid": 1,
          "um": null,
          "update": [Function],
          "vnode": [Circular],
          "withProxy": null,
        },
        "ctx": [Circular],
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <article
          class="linecard borders m-hide"
          data-v-72aaa15f=""
        >
          <h2
            class="linecard__header-text"
            data-v-72aaa15f=""
          >
            bulletin.header
          </h2>
          <form
            action="#"
            class="linecard__form"
            data-v-72aaa15f=""
          >
            <label
              class="linecard__form-label"
              data-v-72aaa15f=""
              for="email"
            >
              bulletin.label
            </label>
            <input
              class="linecard__form-select bulletin__input"
              data-v-72aaa15f=""
              id="email"
              name="email"
              placeholder="bulletin.placeholder"
              required=""
              type="email"
            />
            <button
              class="linecard__form-submit cursor-pointer"
              data-v-72aaa15f=""
              type="submit"
            >
              bulletin.submit
            </button>
          </form>
        </article>,
        "key": null,
        "patchFlag": 0,
        "props": {
          "ref": "VTU_COMPONENT",
        },
        "ref": {
          "f": false,
          "i": [Circular],
          "k": undefined,
          "r": "VTU_COMPONENT",
        },
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": {
          "__file": "/home/andrew/Code/radiomag/radiomag-client-vue/src/components/BulletinComponent.vue",
          "__scopeId": "data-v-72aaa15f",
          "components": {},
          "render": [Function],
        },
      },
      "suspense": null,
      "suspenseId": 0,
      "type": {
        "name": "VTU_ROOT",
        "render": [Function],
        "setup": [Function],
      },
      "u": null,
      "uid": 0,
      "um": null,
      "update": [Function],
      "vnode": {
        "__v_isVNode": true,
        "__v_skip": true,
        "anchor": null,
        "appContext": {
          "app": [Circular],
          "components": {
            "transition": {
              "name": "transition",
            },
            "transition-group": {
              "name": "transition-group",
            },
          },
          "config": {
            "compilerOptions": {},
            "errorHandler": undefined,
            "globalProperties": {},
            "isNativeTag": [Function],
            "optionMergeStrategies": {},
            "performance": false,
            "warnHandler": undefined,
          },
          "directives": {},
          "emitsCache": WeakMap {},
          "mixins": [
            {
              "beforeCreate": [Function],
            },
          ],
          "optionsCache": WeakMap {},
          "propsCache": WeakMap {},
          "provides": {},
          "reload": [Function],
        },
        "children": null,
        "component": [Circular],
        "ctx": null,
        "dirs": null,
        "dynamicChildren": null,
        "dynamicProps": null,
        "el": <article
          class="linecard borders m-hide"
          data-v-72aaa15f=""
        >
          <h2
            class="linecard__header-text"
            data-v-72aaa15f=""
          >
            bulletin.header
          </h2>
          <form
            action="#"
            class="linecard__form"
            data-v-72aaa15f=""
          >
            <label
              class="linecard__form-label"
              data-v-72aaa15f=""
              for="email"
            >
              bulletin.label
            </label>
            <input
              class="linecard__form-select bulletin__input"
              data-v-72aaa15f=""
              id="email"
              name="email"
              placeholder="bulletin.placeholder"
              required=""
              type="email"
            />
            <button
              class="linecard__form-submit cursor-pointer"
              data-v-72aaa15f=""
              type="submit"
            >
              bulletin.submit
            </button>
          </form>
        </article>,
        "key": null,
        "patchFlag": 0,
        "props": null,
        "ref": null,
        "scopeId": null,
        "shapeFlag": 4,
        "slotScopeIds": null,
        "ssContent": null,
        "ssFallback": null,
        "staticCount": 0,
        "suspense": null,
        "target": null,
        "targetAnchor": null,
        "transition": null,
        "type": {
          "name": "VTU_ROOT",
          "render": [Function],
          "setup": [Function],
        },
      },
      "withProxy": null,
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": {
      "compilerOptions": {},
      "errorHandler": undefined,
      "globalProperties": {},
      "isNativeTag": [Function],
      "optionMergeStrategies": {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.2.47",
  },
  "__setProps": [Function],
  "cleanUpCallbacks": [
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
  ],
  "componentVM": {
    "$t": [Function],
  },
  "isDisabled": [Function],
  "rootVM": {
    "$t": [Function],
    "VTU_COMPONENT": {
      "$t": [Function],
    },
  },
  "wrapperElement": <article
    class="linecard borders m-hide"
    data-v-72aaa15f=""
  >
    <h2
      class="linecard__header-text"
      data-v-72aaa15f=""
    >
      bulletin.header
    </h2>
    <form
      action="#"
      class="linecard__form"
      data-v-72aaa15f=""
    >
      <label
        class="linecard__form-label"
        data-v-72aaa15f=""
        for="email"
      >
        bulletin.label
      </label>
      <input
        class="linecard__form-select bulletin__input"
        data-v-72aaa15f=""
        id="email"
        name="email"
        placeholder="bulletin.placeholder"
        required=""
        type="email"
      />
      <button
        class="linecard__form-submit cursor-pointer"
        data-v-72aaa15f=""
        type="submit"
      >
        bulletin.submit
      </button>
    </form>
  </article>,
}
`;
